beacon-chain/powchain/service.go

```diff
@@ -496,6 +530,22 @@ func (w *Web3Service) processPastLogs() error {
 	return nil
 }
 
+func (w *Web3Service) initFromDB() error {
+	powDepositState, err := w.beaconDB.DepositState()
+	if err != nil {
+		return err
+	}
+	if powDepositState == nil {
+		return nil
+	}
+	w.blockHeight = new(big.Int).SetUint64((*powDepositState).LastBlockHeight)
```

Comment:
Again no need to dereference the pointer here and have brackets around it , 
```go
powDepositState.LastBlockHeight
```
the above is fine, same for all the vars below